scripts/services/getConfig.ts

```diff
@@ -116,7 +116,7 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
             "type": "boolean",
             "name": "Disable Dandelion++",
```

Comment:
It's a bit confusing to have config options to "disable" options by enabling them.  If this is how monerod does it, then it makes sense to stay uniform, but the description can probably be a bit more helpful